examples/language-analyzer.go

```diff
@@ -6,10 +6,11 @@ import (
 	"io"
 	"os"
 
+	"gopkg.in/src-d/lookout-sdk.v0/pb"
```

Comment:
I asked in a commet above to describe why we need to change the limit above. I see changes but they don't describe the reason. As I mentioned: the only reason is UAST. For a 1mb file it can be huge.
it means we *need* it only in data server but we *can* do it anywhere.

lookoutd uses an internal helper. That helper sets logging, will set metadata and so on. It also changes the limit if I'm not mistaken because of dummy analyzer. Dummy uses the same helper.